ai/core/src/unitycatalog/ai/core/databricks.py

```diff
@@ -210,15 +205,10 @@ def __init__(
         Args:
             client: The databricks workspace client. If it's None, a default databricks workspace client
                 is generated based on the configuration. Defaults to None.
-            warehouse_id: The warehouse id to use for executing functions. This field is
-                not needed if serverless is enabled in the databricks workspace. Defaults to None.
             profile: The configuration profile to use for databricks connect. Defaults to None.
         """
         self.client = client or get_default_databricks_workspace_client(profile=profile)
-        self.warehouse_id = warehouse_id
```

Comment:
Since we've supported this in previous version, could we raise a warning when "warehouse_id" exists in kwargs and guide customers about how to use serverless compute (with link to documentation)? Otherwise this breaking change should annoy customers :(